api4/user.go

```diff
@@ -693,6 +693,20 @@ func getUsers(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.SetInvalidURLParam("inactive")
 	}
 
+	roleNamesAll := []string{}
+	// MM-47378: validate 'role' related parameters
+	if role != "" || rolesString != "" || channelRolesString != "" || teamRolesString != "" {
```

Comment:
Question: Do you validate `role` as well? It looks like you validate the other three, but not `role`.

Reply:
Good catch!

I was apprehensive of adding this because a role check didn't exist before, not even something like `model.CleanRoleNames(strings.Split(channelRolesString, ","))`.

Let me add one for `role` and see if any tests fail.

Reply:
All the tests pass, thanks for bringing this up.